mf4-csv.py

```diff
@@ -75,58 +67,33 @@ def file_convert():
         #     with open(file+'.ChannelGroup_' + str(x) + '.csv') as f:
         #         first_line = f.readline()
     else:
-        print("You fool; thats not a valid file! The file must be an .mf4")
+        print("You fool; thats not a valid file! The file must be a .mf4")
     handle_data(files)
 
+
+#Streams data into the DB 
```

Comment:
Half your comments have a space and the other half don't. I don't care which you do as long as you're consistent about it
```suggestion
# Streams data into the DB 
```

Reply:
Fixed <3